include/openssl/core.h

```diff
@@ -77,40 +77,40 @@ struct ossl_param_st {
     unsigned int data_type;      /* declare what kind of content is in buffer */
     void *buffer;                /* value being passed in or out */
     size_t buffer_size;          /* buffer size */
+    size_t bn_size;              /* Size of converted BIGNUM */
```

Comment:
No.  This complicates things unnecessarily

Reply:
I agree with @levitte here. I don't see the need for this.